src/Exceptionless.Web/Program.cs

```diff
@@ -73,10 +75,14 @@ public class Program {
                     s.AddSingleton(settings);
                 })
                 .UseStartup<Startup>();
-
-            if (useApplicationInsights)
+				
+			if (useApplicationInsights)
                 builder.UseApplicationInsights(Settings.Current.ApplicationInsightsKey);
 
+            if (settings.EnableMetricsReporting && settings.MetricsConnectionString is DefaultMetricsConnectionString) {
+                settings.MetricsConnectionString = MetricsConnectionString.Parse(settings.MetricsConnectionString.ConnectionString);
```

Comment:
Not sure if we need this check to see if it's the default as it it's the main entry point and doesn't hurt anything to parse It twice if it's by accident.